lib/galaxy/webapps/galaxy/api/history_contents.py

```diff
@@ -511,22 +511,22 @@ def download_dataset_collection(
     @router.post(
         "/api/histories/{history_id}/contents/dataset_collections/{id}/prepare_download",
         summary="Prepare an short term storage object that the collection will be downloaded to.",
+        include_in_schema=False,
+    )
+    @router.post(
```

Comment:
I have an alternate fix for this... in a branch that will be very conflicted 😆. hackathons